apps/mobile/src/components/LaunchPad/UserShare/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FlatList, View } from 'react-native';
+
+import { useStyles } from '../../../hooks';
+import { UserShareInterface } from '../../../types/keys';
+import { Text } from '../../Text';
+import stylesheet from './styles';
+import Loading from '../../Loading';
+
+export type UserShareProps = {
+  loading: boolean;
+  shares?: UserShareInterface[];
+};
+
+export const UserShare: React.FC<UserShareProps> = ({ shares, loading }) => {
+  const styles = useStyles(stylesheet);
+
+  return loading ? (
+    <Loading />
+  ) : (
+    <FlatList
+      data={shares}
+      keyExtractor={(item, index) => index.toString()}
+      renderItem={({ item }) => (
+        <View style={[styles.container, styles.borderBottom]}>
+          <View style={styles.borderBottom}>
+            <Text fontSize={14} weight="semiBold">
+              Memecoin Address
+            </Text>
+            <Text fontSize={14}>{item.token_address}</Text>
+          </View>
+
+        <View style={styles.borderBottom}>
+            <Text fontSize={14} weight="semiBold">
+              Owner Address:
+            </Text>
+            <Text fontSize={14}>{item.owner}</Text>
+          </View> 
+
+          <View style={styles.borderBottom}>
+            <Text fontSize={14} weight="semiBold">
+              Price
```

Comment:
Amount, coin received, type of the tx are needed here.. 

Owner address is not needed, because it's every time our address no?